05-extra.md

```diff
@@ -0,0 +1,225 @@
+---
+layout: page
+title: Version Control with Git
+subtitle: Additional Topics
```

Comment:
Where can I look up what should go in the `minutes` field? I don't see it in any other pages and it's not mentioned in [Jekyll's documentation on front matter](http://jekyllrb.com/docs/frontmatter/).